deltacat/utils/pyarrow.py

```diff
@@ -383,12 +384,15 @@ def s3_file_to_table(
             f"and content_type={content_type}"
         )
 
-        parquet_reader_func = s3_parquet_file_to_table
+        # TODO(sammy): pass row groups to daft reader as well.
+        # We currently take in the row offset and len rather than the indices of the requested row groups.
+        # Should be a simple change to have the option for both
+        parquet_reader_func = daft_s3_file_to_table
```

Comment:
Can we check "reader_type: ["daft", "pyarrow"]" in the kwargs returned by  `pa_read_func_kwargs_provider(content_type, {})`?

Reply:
Went ahead and updated it. let me know if it's what you had in mind?